libraries/splunk_app.rb

```diff
@@ -266,6 +273,18 @@ def install_from_tar(filename, expected_version, installed_version)
         tarfile.close if tarfile
       end
 
+      def manage_lookups
+        lookups = new_resource.lookups
+        lookups.each do |file_name, url|
+          if url.nil?
```

Comment:
Updated here: 754b5a382f2dbe1a5f513216edd31120fa37da4c

Also updated the check for file extension.